core/services/relay/evm/config_poller.go

```diff
@@ -86,10 +84,20 @@ func makeConfigSetMsgArgs(withFeedID bool) abi.Arguments {
 			Name: "signers",
 			Type: utils.MustAbiType("address[]", nil),
 		},
-		{
+	}
+
+	if withFeedID {
+		args = append([]abi.Argument{configSetFeedIDArg}, args...)
+	} else {
+		// We only support `transmitters` when not using feedId
+		transmittersArg := abi.Argument{
 			Name: "transmitters",
 			Type: utils.MustAbiType("address[]", nil),
-		},
+		}
+		args = append(args, transmittersArg)
```

Comment:
This is a bit of a mess, can we lean on the generated wrapper log struct and unmarshal methods instead of doing this manually?